frontend/src/components/Items.vue

```diff
@@ -127,13 +142,43 @@ export default {
       },
       immediate: true,
     },
+    items: {
+      handler: function () {
+        if (
+          this.items.some((item) => item.hasOwnProperty("activeState")) &&
+          this.mode === 'single'
+        ) {
+          let activeItem = null;
+          for (let i = 0; i < this.items.length; i++) {
```

Comment:
It looks as though this loop could be replaced with a simple `.find`
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find